docs/quickstart.rst

```diff
@@ -1,6 +1,166 @@
 Quickstart
 ==========
 
+Welcome to the *Septentrion* quickstart documentation.
+
+In this tutorial, you will learn how to write migrations in the expected format, how to visualize them and how to run
+them against a PostgreSQL database.
+
+Prepare your database
+---------------------
+
+If you already have a PostgreSQL database around, make sure to note the connection
+parameters. Otherwise, we'll create one together with Docker_:
+
+.. _Docker: https://docs.docker.com/
+
+.. code-block:: console
+
+    $ docker run --detach --rm -p 5432:5432 --name septentrion_db postgres
+
+.. note::
+
+    If you need to stop the docker at some point, use ``docker stop septentrion_db``.
+
+You'll also need ``psycopg2``, which is notoriously complex to install. Procrastinate
+will install the ``psycopg2`` python package, but will expect the system to already
+have the prerequisites (on ``Ubuntu``)::
+
+    $ sudo apt install libpq-dev python-dev postgresql-client
+
+.. note::
+    On a different OS, if you experiment difficulties, we'll be grateful if you can tell
+    us via an issue so that we improve this documentation.
+
+Install and configure Septentrion
+---------------------------------
+
+Within a virtualenv_, install Septentrion with:
+
+.. _virtualenv: https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments
+
+.. code-block:: console
+
+    (venv) $ pip install septentrion
+
+Next we will configure the connection to the PostgreSQL database. We can do this either with command line flags,
+environment variables or a configuration file.
+In this tutorial, we will use a configuration file, ``septentrion.ini``.
+
+.. code-block:: ini
+
+    [septentrion]
+    host=localhost
+    # port=5432
+    # username=postgres
+
+.. note::
+
+    With the Docker setup described above, you should be good to go.
+    If you need additional configuration parameters to connect to your database, have a look at
+    :ref:`advanced configuration options <how-to/configure>`.
+
+
+Write migrations
+----------------
+
+Migrations are SQL files that are executed in order. Migrations are grouped together in successive versions.
+All migrations in the same version are placed in a folder named after the version number.
+
+Let's create our migration folder, and a first version folder.
+
+.. code-block:: console
+
+    $ mkdir migrations
+    $ mkdir migrations/1.0
```

Comment:
The default value of the setting `migration_root` is `'.'`. We have to add 'migrations_root=migrations' in the `setting.ini` file or create migrations directly on the root dir of this tutorial project.

Reply:
Updated `septentrion.ini`